kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java

```diff
@@ -106,7 +106,7 @@
     private String awayMessageTextColor = "#A9A4A4";
     private boolean isAgentApp = false;
     private boolean hideGroupSubtitle = false;
-    private boolean disableGlobalStoragePermission = false;
+    private boolean disableGlobalStoragePermission = true;
```

Comment:
can we change it to the other globalGlobalStoragePermission true or false.
its easy to understand something as:
enable: true vs disable: false  - although both means the same thing

Reply:
This setting will not be needed now, we will remove this from docs. The storage permission is not needed globally when we have a better implementation of checking them on demand. Thats why we are setting it as true internally so that users don't have to change it.